### PR TITLE
attesation-tool/ecdsa: include ppid in output

### PR DESCRIPTION
**Pull request by ptrus**
Include PPID in the output of the DCAP attestation. Useful for detecting cpu change via the attesttion tool.

### CODE REVIEW

File: attestation-tool/src/ecdsa.rs [hunk 1]
```diff
@@ -341,6 +359,7 @@ pub fn try_ecdsa(aesm_client: &AesmClient, loader: &mut IsgxDevice) -> Result<TC
         "TCB evaluation data number: {:?}",
         tcb.tcb_evaluation_data_number
     );
+    println!("PPID: {:?}", hex::encode(ppid.unwrap()));
```
Comment (kostko) [inline attestation-tool/src/ecdsa.rs:362]: Maybe unwrap above with others?

**Review by kostko — APPROVED** [review]
> Useful for detecting cpu change via the attesttion tool.

Note that this probably doesn't imply CPU change as it can also happen in case of TCB recovery AFAIK.

**Comment by ptrus** [comment]
>


> > Useful for detecting cpu change via the attesttion tool.
> 
> Note that this probably doesn't imply CPU change as it can also happen in case of TCB recovery AFAIK.

Based on the description from the documentation I assumed it wont:
> Provisioning ID for a processor package or platform instance. PPID is not TCB dependent

But didn't test it out.

**Comment by kostko** [comment]
Ah ok, I guess it is then just the encrypted blob that changes.

**Pull request merged**